GVFS/GVFS.Common/Git/IGitInstallation.cs

```diff
@@ -4,5 +4,6 @@ public interface IGitInstallation
     {
         bool GitExists(string gitBinPath);
         string GetInstalledGitBinPath();
+        string GetGVFSHooksPath();

```

Comment:
Path to `GVFS.Hooks` is needed by `GetActiveRepoListHandler::IsValidRepo()`. Not sure if `IGitInstallation` interface is the right place for `GetGVFSHooksPath`. Share your thoughts.

Reply:
Right. I didn't see it was there. Thank you Kevin.